visual_behavior/data_access/loading.py

```diff
@@ -121,6 +121,39 @@ def get_ophys_glm_dir():
     return r'//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/ophys_glm'
 
 
+def get_stimulus_response_df_dir(interpolate=True, output_sampling_rate=30, event_type='all'):
+    base_dir = r'//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/platform_paper_cache/stimulus_response_dfs'
+    if interpolate:
+        save_dir = os.path.join(base_dir, event_type, 'interpolate_' + str(output_sampling_rate) + 'Hz')
+    else:
+        save_dir = os.path.join(base_dir, event_type, 'original_frame_rate')
+    if not os.path.exists(save_dir):
+        os.mkdir(save_dir)
+    return save_dir
+
+
+def get_multi_session_df_dir(interpolate=True, output_sampling_rate=30, event_type='all'):
+    base_dir = get_platform_analysis_cache_dir()
+    if interpolate:
+        save_dir = os.path.join(base_dir, 'multi_session_mean_response_dfs', 'interpolate_' + str(output_sampling_rate) + 'Hz')
+    else:
+        save_dir = os.path.join(base_dir, 'multi_session_mean_response_dfs', 'original_frame_rate')
+    if not os.path.exists(save_dir):
+        os.mkdir(save_dir)
+    return save_dir
+
+
+# def get_multi_session_df_dir(interpolate=True, output_sampling_rate=30, event_type='all'):
```

Comment:
seems like this is repeated function. Should be deleted?

Reply:
actually no, i want to keep the old version here for my own memory. i have two different versions of the multi_session_df cache currently, but i will delete this old version once i finalize and consolidate